deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: druid-operator
           # Replace this with the built image name
-          image: datainfrahq/druid-operator:latest
+          image: 164105964448.dkr.ecr.eu-central-1.amazonaws.com/struss-test-druid-operator:latest
```

Comment:
change it back? :)

Reply:
dang.... i thought i found all "internal references". Definitely changing this back